src/main/java/ch/admin/bag/covidcertificate/api/request/conversion/ConversionReason.java

```diff
@@ -0,0 +1,12 @@
+package ch.admin.bag.covidcertificate.api.request.conversion;
+
+public enum ConversionReason {
+
+    U18_CONVERSION("U18");
```

Comment:
Die Umwandlung betrifft gemäss neuer Definition alle Impfzertifikate die in den nächsten 3 Monaten ablaufen und nicht nur U18 (siehe https://confluence.bit.admin.ch/pages/viewpage.action?pageId=363205451). Daher wäre es aus meiner Sicht sinnvoll, den ConversionReason anzupassen. z.B. auf "TECH_EXPIRING_VACCINATION_CONVERSION" oder so?

Reply:
Ich habe VACCINATION_CONVERSION gewählt und den reason 1:1 also nicht abgekürzt gesetzt.